src/mechanics/tools/AdaptiveSolve.cpp

```diff
@@ -54,6 +57,16 @@ void AdaptiveSolve::Solve(double tEnd)
                                 "Time step smaller than prescribed minimal time step " + std::to_string(dtMin) + ".");
             continue; // without updating the global time
         }
+        catch (Exception& e)
+        {
+            std::cout << e.what() << std::endl;
+            return;
+        }
+        catch (std::exception& e)
```

Comment:
Imho only one of these `catch`es is needed. Since `NuTo::Exception` inherits from `std::exception`, both will be caught by the lower one and are handled the same way anyway. BTW, why are you catching these exceptions anyway?

Reply:
Even the documentation of the method says that it only catches `NoConvergence`. So I forgot to update that. Fixed now.